components/schemes/Typography/tests/unit/MDCTypographySchemeTests.m

```diff
@@ -44,4 +44,26 @@ - (void)testInitializer {
   XCTAssertEqual(latestScheme.overline, scheme201804.overline);
 }
 
+- (void)testTypographySchemeCopyTest {
+  // Given
+  MDCTypographyScheme *typographyScheme = [[MDCTypographyScheme alloc] init];
+  MDCTypographyScheme *typographySchemeCopy = [typographyScheme copy];
+
+  // Then
+  XCTAssertFalse(typographyScheme == typographySchemeCopy);
+  XCTAssertEqual(typographyScheme.headline1, typographySchemeCopy.headline1);
```

Comment:
Please use XCTAssertEqualObjects since we're ok with copied fonts, and don't want to fail if they aren't the same object.

Reply:
Done.